web/src/services/TestRun.service.ts

```diff
@@ -39,6 +42,59 @@ const TestRunService = () => ({
 
     return eventsWithoutPolling;
   },
+
+  getAnalyzerErrorsHashedBySpan(linterResult: LinterResult): TAnalyzerErrorsBySpan {
+    return linterResult.plugins
```

Comment:
What do you think about moving this logic to service functions?

Reply:
Hey @xoscar thanks for the review. What do you mean by service functions? This is already part of the TestRun service.

Reply:
What haha I thought it was part of the selectors, sorry that was a mistake